src/main/java/uk/gov/pay/api/service/ConnectorService.java

```diff
@@ -43,6 +46,22 @@ public Charge getCharge(Account account, String paymentId) {
         throw new GetChargeException(response);
     }
 
+    public PaymentEvents getChargeEvents(Account account, String paymentId) {
+        Response connectorResponse = client
+                .target(connectorUriGenerator.chargeEventsURI(account, paymentId))
+                .request()
+                .accept(MediaType.APPLICATION_JSON)
+                .get();
+
+        if (connectorResponse.getStatus() == SC_OK) {
+            PaymentEvents response = connectorResponse.readEntity(PaymentEvents.class);
```

Comment:
Not a big thing but we can return response directly here

Reply:
yeah, even IDE told me that, but in the end I forgot to do it. Will update separately.